src/PrestaShopBundle/Twig/Extension/LocalizationExtension.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Twig\Extension;
+
+use DateTime;
+use DateTimeInterface;
+use Twig\Extension\AbstractExtension;
+use Twig_SimpleFilter;
+
+class LocalizationExtension extends AbstractExtension
```

Comment:
@matks wdyt about name?

Also, I think we should document this extension as it is very useful.

Reply:
A bit generic ... but it works for now. We should monitor what else we'll put in it (right now it's a DateFormatterExtension). If we add more and more localization-related functions in it, there it works 👍

Reply:
Yeah, you are right, it's a little generic, maybe we can rename it to `DateFormatterExtension` instead? Now when I think about it, it wouldn't make sense to inject multiple services, if we need more, lets say for price display, we could better create `PriceFormatterExtension`.

Reply:
It makes sense to have multiple services if they are part of the same component. See `PrestaShopBundle\Twig\Extension\GridExtension` 😉 

Price formats and date formats are both linked to localization so I think it's alright to have them side by side.

Reply:
(it's unlikely that you'd need to localize dates but not prices)